lib/uplink/bucket.go

```diff
@@ -29,6 +29,9 @@ type Bucket struct {
 
 // TODO: move the object related OpenObject to object.go
 
+// Info returns information about the bucket.
+func (b *Bucket) Info() storj.Bucket { return b.bucket }
```

Comment:
any field we need here should probably be on the bucket type directly? there are already a few

Reply:
`CreateBucket` on the project type already returns a `storj.Bucket` https://github.com/storj/storj/blob/f9ed0dc1a83c0b7e6debef465ac846af56192885/lib/uplink/project.go#L97

We could promote up just the `Attribution` field, but that seems silly to do just for a test. I'm just going to work around this with some reflect magic, for now.